File: src/com/trapps/jvsk/coursera/control_structures/CS.java
package com.trapps.jvsk.coursera.control_structures;

import java.io.IOException;

/**
 * @author Akash Patra
 */
public class CS {

    public static void main(String[] args) {
        exceptionTest();
    }

    private static void exceptionTest() {
        try {
            ExceptionsKt.foo();
        } catch (IOException e) {

        }
    }
}

File: src/in/co/trapps/jvsk/coursera/extensions/CS.kt
package `in`.co.trapps.jvsk.coursera.extensions

/**
 * @author Akash Patra
 */
fun main(args: Array<String>) {
    getLastButOneChar()
}

/**
 * Import is reqd to access Extension Function, as it is different file
 */
fun getLastButOneChar() {
    val data = "abc"
    println("LastButOneChar of $data is ${data.lastButOneChar()}")
}